contact.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
- [ ] I kindly suggest that you move your contact.html syntax to your index.html because your portfolio is supposed to be one page with one html file.